.github/workflows/backport.yml

```diff
@@ -1,16 +1,28 @@
 name: Backport
 on:
-  pull_request:
+  pull_request_target:
     types:
       - closed
       - labeled
 
 jobs:
   backport:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
     name: Backport
     steps:
+      - name: GitHub App token
+        id: github_app_token
+        uses: tibdex/github-app-token@v1.5.0
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+          installation_id: 22958780
+
       - name: Backport
-        uses: tibdex/backport@v1
+        uses: VachaShah/backport@v1.1.4
```

Comment:
I am not sure how the backport workflow works. Does this needs to tied to a user ?

Reply:
This is not a user. It is using a GitHub Action to provide this workflow.

https://github.com/VachaShah/backport

The other OpenSearch projects are using this as well. You can see the links I shared in the description.